frontend/ui/data/css_tweaks.lua

```diff
@@ -192,6 +192,21 @@ h1, h2, h3, h4, h5, h6 { hyphens: none !important; }
     },
     {
         title = _("Paragraphs"),
+        {
+            id = "paragraph_web_browser_style";
+            title = _("Generic web browser paragraph style"),
+            description = _("Display paragraphs as web browsers do, discarding KOReader's book paragraph style.\nThis might be needed with some documents that expect this style as the default, and only use CSS when it needs to diverge from this default."),
```

Comment:
I'm not sure if "as web browsers do" is very clear. I'd just call it something like full-block style paragraphs vs indented paragraphs.

Reply:
That's not just that: it's margin top and bottom (I saw that some publishers use `margin-bottom: -1em` to revert that... instead of using a simpler `margin-top: 0 margin-bottom: 0`), no text justification (so, `text-align: left`), no text-indent ... Just as we are used to see `<P>` in all web browsers :)
And setting this combo make some publisher styles just display nice.

Reply:
What do you mean by also margin top and bottom? I meant,

Full-block style  (block-style is standard English for this concept)
```
I am a paragraph in a modern American business letter.

I am another paragraph. I'm not indented and there's whitespace
in between me and my predecessor.
```

Indented
```
I probably require no explanation,
but I am a paragraph
in a traditional Dutch letter.
    I, as a second paragraph, am indented.
```

Knowing what browsers do if you just give them some plain HTML without custom styling is more specialized knowledge, I'd say.

But it might work by just mixing the explanation in there a bit? `Display paragraphs as browsers do, in full-block style without indentation or justification, discarding…`

Reply:
> it might work by just mixing the explanation in there a bit

Indeed, that will do.